Source/Core/InputCommon/GCAdapter.cpp

```diff
@@ -322,14 +332,7 @@ static bool CheckDeviceAccess(libusb_device* device)
     return false;
   }
 
-  ret = libusb_claim_interface(s_handle, 0);
```

Comment:
It looks like the if block just above this one ("this split is needed so that we don't avoid claiming the interface when...") should also be moved.

Reply:
Gotcha, will update and push. Currently away from my desktop so will take a bit to get here.

Reply:
In the latest commit, I've moved that condition check to immediately after the kernel detach would happen. This should guard the interface claim from being avoided like the comment describes, unless I misread something.

I'm buildtesting as I speak, since the bot doesn't autotrust :)

Reply:
After latest commit, the build goes fine on my end. I assume that where I've moved this to will work, so I'll resolve, but if it needs to be someplace else feel free to drop another comment.